packages/xo-web/src/xo-app/proxies/index.js

```diff
@@ -56,6 +56,13 @@ const ACTIONS = [
 ]
 
 const INDIVIDUAL_ACTIONS = [
+  {
+    collapsed: true,
+    disabled: ({ url }) => url === undefined,
+    handler: ({ url }) => navigator.clipboard.writeText(url),
+    icon: 'clipboard',
+    label: _('proxyCopyUrl'),
```

Comment:
Would it make sense to show the URL in the tooltip? I usually like to know what's going to be copied to my clipboard and that's what we do for "copy UUID" buttons, but I don't know if it's necessary here.

/cc @julien-f https://github.com/vatesfr/xen-orchestra/pull/6287#discussion_r896464346

Reply:
Do you mean displaying the URL and have a copy button?
In a dedicated column?

We (I) decided not to do it because the URL can be quite long (especially due to the token) and copying the URL is a very rare action.